src/utils/fail_point_impl.h

```diff
@@ -47,6 +47,7 @@ struct fail_point
         Off,
         Return,
         Print,
+        Void,
```

Comment:
How about adding some comments for the fail_point::task_type to describe the difference between them and how to use it?

Reply:
done